lib/colorjoe.js

```diff
@@ -86,6 +86,7 @@ var drag = (function() {
         if(klass) {
           elem.className = klass;
         }
+        if(p !== null && p !== "undefined")
```

Comment:
use { with ifs }

also, use p && typeof p !== "undefined" as the boolean expression